src/Microsoft.AspNetCore.Server.IIS/HttpContextExtensions.cs

```diff
@@ -1,50 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Http.Features;
-
-namespace Microsoft.AspNetCore.Server.IIS
-{
-    /// <summary>
-    /// Extensions to <see cref="HttpContext"/> that enable access to IIS features.
-    /// </summary>
-    public static class HttpContextExtensions
-    {
-        /// <summary>
-        /// Gets the value of a server variable for the current request.
-        /// </summary>
-        /// <param name="context">The http context for the request.</param>
-        /// <param name="variableName">The name of the variable.</param>
-        /// <returns>
-        /// <c>null</c> if the feature does not support the <see cref="IServerVariablesFeature"/> feature.
-        /// May return null or empty if the variable does not exist or is not set.
-        /// </returns>
-        /// <remarks>
-        /// For a list of common server variables available in IIS, see http://go.microsoft.com/fwlink/?LinkId=52471.
-        /// </remarks>
-        public static string GetIISServerVariable(this HttpContext context, string variableName)
```

Comment:
@shirhatti are you cool with this? I was under the impression you wanted these to be more visible.

Reply:
I’d hate for this to show up by default in intellisense. These are referenced by default.

Reply:
That's why the namespace was `Microsoft.AspNetCore.Server.IIS`, that won't show up in Startup.

Reply:
Then that’s fine imo, no need to delete.  Though it kinda forces user code to check for null.

Reply:
Why have extra API that almost no-one uses? How is it different from other feature we don't write extension methods for?

Reply:
We went over this when we added the method the first time. Nothing has changed, I don't see any reason to rehash the discussion now.